src/components/DownloadButton/DownloadButton.js

```diff
@@ -0,0 +1,14 @@
+import React, { PropTypes, PureComponent } from 'react'
+import './DownloadButton.scss'
+
+export default class DownloadButton extends PureComponent {
+  render () {
+    return (
+      <a className='download-button' target='_blank' href={this.props.url}>Download</a>
```

Comment:
Please add `rel="noopener noreferrer"`, see [this article](https://mathiasbynens.github.io/rel-noopener/).